### ISSUE TITLE
where can I find the training dataset?

### ISSUE THREAD

**Issue by mundher** [Spidy20/face_mask_detection#8]
could you please provide us the dataset which has been used to train your model?

**Comment by Spidy20** [comment]
I created my own dataset, I mentioned all the things here
Watch this tutorial playlist:-
https://www.youtube.com/playlist?list=PLsT53VV2LIIGiJOGJk-jCZ1q6t03x_R8_

**Comment by Harsh-Vavaiya** [comment]
@Spidy20 I didn't find any dataset link your youtube playlist. Please, share labelled dataset.

**Comment by Spidy20** [comment]
I shown how you can create the dataset,  I have not shared my created
dataset.

**Comment by Harsh-Vavaiya** [comment]
I know how to do this. I just wanted to see your dataset.

**Comment by Xx-Ashutosh-xX** [comment]
Could you please provide us with the dataset. It will benefit us all.

**Comment by Spidy20** [comment]
https://youtu.be/eq4BDzdax08

Here it is, video of dataset generation ✌️

**Comment by Harsh-Vavaiya** [comment]
@Xx-Ashutosh-xX Man, I think this man didn't understand or he used someone else dataset although he didn't sharing.

And sharing some weird tutorial. everyone who got into in this field, very first they know how dataset come into existence.

**Comment by Xx-Ashutosh-xX** [comment]
@mundher @Harsh-Vavaiya 
Everyone can go here, I have included a dataset download link 
https://github.com/Xx-Ashutosh-xX/Real-Time-Face-Mask-Detection-Model

**Comment by Spidy20** [comment]
@Harsh-Vavaiya  You can see I provided a tutorial on how you can create a dataset, it takes 2 min to create your own dataset, that's why I am not sharing my dataset. I did this face mask detection when it was not trending and that time any dataset was not available. Check out here how I created my own dataset:- https://youtu.be/eq4BDzdax08

I used TF Obj API in my tutorial, I don't know what is the mentality of a people kind like you. who didn't appreciate someone's efforts. anyway, you can check I did face mask detection on the around 20th April, when people don't have an idea about it. 
Check out my Linkedin post link:- https://www.linkedin.com/posts/kushal-bhavsar_tensorflow-deeplearning-pytorch-activity-6657483656447258624-1pgy

**Comment by Harsh-Vavaiya** [comment]
@Spidy20 very good work man that you know what people are thinking and people are doing. Anyway I was doing same thing in April but due to it's products of some company I can't revel to social media.

Thank you man for sharing dataset.

**Comment by Xx-Ashutosh-xX** [comment]
> @Harsh-Vavaiya You can see I provided a tutorial on how you can create a dataset, it takes 2 min to create your own dataset, that's why I am not sharing my dataset. I did this face mask detection when it was not trending and that time any dataset was not available. Check out here how I created my own dataset:- https://youtu.be/eq4BDzdax08
> 
> I used TF Obj API in my tutorial, I don't know what is the mentality of a people kind like you. who didn't appreciate someone's efforts. anyway, you can check I did face mask detection on the around 20th April, when people don't have an idea about it.
> Check out my Linkedin post link:- https://www.linkedin.com/posts/kushal-bhavsar_tensorflow-deeplearning-pytorch-activity-6657483656447258624-1pgy

@Spidy20  Good for you,  But you see the problem in this field is finding a good dataset when a new problem arises. So if in future someone (new to this) is trying to gain some experience it could really help them.
But thanks anyway

**Comment by Spidy20** [comment]
@Xx-Ashutosh-xX  Yes, finding a dataset is a problem in our field. Now there are lots of datasets available for face mask detection. but when I did this there was no dataset available. and you know sometimes sharing your work is harmful to us. Because if we share our hard work to openly, people publish it without giving our reference. Just you see when Pyimagesearch uploaded the tutorial. After 1 week I have seen lots of people doing Face mask detection. but no one mentions them. They are just describing their work is completed by themselves without any help. But they copied the full code of Pyimagesearch without understanding the logic. @Harsh-Vavaiya I think you will also agree on this.

**Comment by Spidy20** [comment]
I already doing many things for our community, Here is my youtube channel and Instagram community where I am giving them knowledge of AI/ML in free.
https://www.youtube.com/c/MachineLearningHub
https://www.instagram.com/machine_learning_hub.ai/
